<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SRMainWindow</class>
 <widget class="QMainWindow" name="SRMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spatial Ressection</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionSet"/>
    <addaction name="actionMove"/>
    <addaction name="actionZoom"/>
    <addaction name="separator"/>
    <addaction name="actionFit"/>
    <addaction name="actionInsert_Points"/>
   </widget>
   <widget class="QMenu" name="menu_Execute">
    <property name="title">
     <string>&amp;Execute</string>
    </property>
    <addaction name="actionFlight"/>
    <addaction name="actionSpatialRessection"/>
    <addaction name="actionTable"/>
   </widget>
   <addaction name="menu_View"/>
   <addaction name="menu_Execute"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Point coordinates</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableView" name="table1"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSet"/>
   <addaction name="actionUnset"/>
   <addaction name="actionMove"/>
   <addaction name="actionZoom"/>
   <addaction name="separator"/>
   <addaction name="actionFit"/>
   <addaction name="actionGrid"/>
   <addaction name="actionInsert_Points"/>
   <addaction name="separator"/>
   <addaction name="actionFlight"/>
   <addaction name="actionSpatialRessection"/>
   <addaction name="actionTable"/>
  </widget>
  <action name="actionZoom">
   <property name="icon">
    <iconset resource="sr_resource.qrc">
     <normaloff>:/Icons/resource/images/find.png</normaloff>:/Icons/resource/images/find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="icon">
    <iconset resource="sr_resource.qrc">
     <normaloff>:/Icons/resource/images/move.png</normaloff>:/Icons/resource/images/move.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Move</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionFit">
   <property name="icon">
    <iconset resource="sr_resource.qrc">
     <normaloff>:/Icons/resource/images/fit.png</normaloff>:/Icons/resource/images/fit.png</iconset>
   </property>
   <property name="text">
    <string>Fit &amp;view</string>
   </property>
   <property name="toolTip">
    <string>Fit view</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSet">
   <property name="icon">
    <iconset resource="sr_resource.qrc">
     <normaloff>:/Icons/resource/images/mark.png</normaloff>:/Icons/resource/images/mark.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Set point</string>
   </property>
   <property name="toolTip">
    <string>Set point</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionUnset">
   <property name="icon">
    <iconset resource="sr_resource.qrc">
     <normaloff>:/Icons/resource/images/unmark.png</normaloff>:/Icons/resource/images/unmark.png</iconset>
   </property>
   <property name="text">
    <string>Unset</string>
   </property>
   <property name="toolTip">
    <string>Unset Point</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionGrid">
   <property name="icon">
    <iconset resource="sr_resource.qrc">
     <normaloff>:/Icons/resource/images/grid.png</normaloff>:/Icons/resource/images/grid.png</iconset>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>Activate Grid</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSpatialRessection">
   <property name="icon">
    <iconset resource="sr_resource.qrc">
     <normaloff>:/Icons/resource/images/exterior.png</normaloff>:/Icons/resource/images/exterior.png</iconset>
   </property>
   <property name="text">
    <string>Exterior &amp;orientation</string>
   </property>
   <property name="iconText">
    <string>Spatial Ressection</string>
   </property>
   <property name="toolTip">
    <string>Exterior orientation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFlight">
   <property name="icon">
    <iconset resource="sr_resource.qrc">
     <normaloff>:/Icons/resource/images/fd.png</normaloff>:/Icons/resource/images/fd.png</iconset>
   </property>
   <property name="text">
    <string>Set &amp;flight</string>
   </property>
   <property name="toolTip">
    <string>Set flight</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="icon">
    <iconset resource="sr_resource.qrc">
     <normaloff>:/Icons/resource/images/table.png</normaloff>:/Icons/resource/images/table.png</iconset>
   </property>
   <property name="text">
    <string>View &amp;report</string>
   </property>
   <property name="toolTip">
    <string>View report</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionInsert_Points">
   <property name="icon">
    <iconset resource="sr_resource.qrc">
     <normaloff>:/Icons/resource/images/export.png</normaloff>:/Icons/resource/images/export.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Insert points</string>
   </property>
   <property name="toolTip">
    <string>Insert points</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="sr_resource.qrc"/>
 </resources>
 <connections/>
</ui>
